assets/js/base/components/checkout/no-shipping/style.scss

```diff
@@ -1,9 +1,19 @@
-.wc-block-checkout__no-shipping .components-placeholder__fieldset {
+.wc-block-checkout__no-shipping {
 	display: block;
-}
+	margin-bottom: $gap !important;
```

Comment:
We could remove the `!important` here increasing the specificity of the selector as follows:
```CSS
.components-placeholder.wc-block-checkout__no-shipping {
```
When possible, I think it's better to increase specificity with extra classes in the selectors than using `!important`, this way it's easier to add styling on top of it if themes want to.